src/analytics.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+const FS = require('fs')
+import { default as OS } from 'os'
+import type Config from 'cli-engine-command/lib/config'
+import HTTP from 'cli-engine-command/lib/http'
+import Netrc from 'netrc-parser'
+import Output from 'cli-engine-command/lib/output'
+
+export default class AnalyticsCommand {
+  command: string
+  plugin: string
+  pluginVersion: string
+  version: string
+  platform: string
+  arch: string
+  start: any
+  analyticsPath: string //= Path.join(CacheHome, "analytics.json")
+  config: Config
+
+  constructor (command: string, plugin: string, pluginVersion: string, config: Config) {
+    this.config = config
+    this.command = command
+    this.plugin = plugin
+    this.pluginVersion = pluginVersion
+    this.version = this.config.config.version
+    this.platform = OS.platform()
+    this.arch = OS.arch()
+    this.analyticsPath = '~/.local/share/heroku/analytics.json'
```

Comment:
@KarateCowboy please do not hard code this, you can get the cache directory from here

https://github.com/heroku/cli-engine-command/blob/master/src/config.js#L52